server/tests/state/completeCallbacks.ts

```diff
@@ -57,11 +56,11 @@ describe('Complete callback tests', () => {
       gameState.players[selectorIndex].responses.push(firstResponse);
       gameState.players[matcherIndex].responses.push(differentResponse);
       gameState.beginSelection();
-      gameState.selectionType = SelectionType.Strike;
+      gameState.selectionType = 'strike';
     });
 
-    it.skip('Selector Disconnects while selecting', (done) => {
-      gameState.registerSelectionUnsuccessfulCb(done);
+    it.skip('Selector Disconnects while selecting', () => {
+      //gameState.registerSelectionUnsuccessfulCb(done);
```

Comment:
Are you keeping this comment here for some reason? Same question for lines 91 and 102

Reply:
The way these callbacks are used has been broken for a while, so leaving them in to document the old expected behavior until a good understanding of what the actual expected behavior comes along